include/z64cutscene_commands.h

```diff
@@ -0,0 +1,419 @@
+    CMD_BBH(continueFlag, roll, frame), CMD_W(viewAngle), CMD_HH(xPos, yPos), CMD_HH(zPos, unused)
+
+/**
+ * ARGS
+ *   s16 unk (u), s16 startFrame (s), s16 endFrame (e)
+ * FORMAT
+ *   Capital U is Unused
+ *   00000008 uuuussss eeeeUUUU
+ *   size = 0xC
+ */
+#define CS_CMD_08_LIST(unk, startFrame, endFrame, unused) \
+    CS_CMD_08, CMD_HH(unk, startFrame), CMD_HH(endFrame, unused)
+
+/**
+ * ARGS
+ *   s8 continueFlag (c), s8 roll (r), s16 frame (f), f32 viewAngle (a), 
+ *   s16 xPos (x), s16 yPos (y), s16 zPos (z)
+ * FORMAT
+ *   Capital U is Unused
+ *   ccrrffff aaaaaaaa xxxxyyyy zzzzUUUU
+ *   size = 0x10
+ */
+#define CS_CMD_08(continueFlag, roll, frame, viewAngle, xPos, yPos, zPos, unused) \
+    CMD_BBH(continueFlag, roll, frame), CMD_W(viewAngle), CMD_HH(xPos, yPos), CMD_HH(zPos, unused)
+
+/**
+ * ARGS
+ *   s32 entries (e)
+ * FORMAT
+ *   00000009 eeeeeeee
+ *   size = 0x8
+ */
+#define CS_CMD_09_LIST(entries) CS_CMD_09, CMD_W(entries)
+
+/**
+ * ARGS
+ *   s16 unk (u), s16 startFrame (s), s16 endFrame (e), s16 unk2 (v), s16 unk3 (w), s16 unk4 (x)
+ * FORMAT
+ *   Capital U is Unused
+ *   uuuussss eeeevvww xxUUUUUU
+ *   size = 0xC
+ */
+#define CS_CMD_09(unk, startFrame, endFrame, unk2, unk3, unk4, unused0, unused1) \
+    CMD_HH(unk, startFrame), CMD_HBB(endFrame, unk2, unk3), CMD_BBH(unk4, unused0, unused1)
+
+/**
+ * ARGS
+ *   s32 cmdType (c), s32 entries (e)
+ * FORMAT
+ *   cccccccc eeeeeeee
+ *   size = 0x8
+ */
+#define CS_UNK_DATA_LIST(cmdType, entries) CMD_W(cmdType), CMD_W(entries)
+
+/**
+ * ARGS
+ *   s32 unk1 (a), s32 unk2 (b), s32 unk3 (c), s32 unk4 (d), s32 unk5 (e), s32 unk6 (f), 
+ *   s32 unk7 (g), s32 unk8 (h), s32 unk9 (i), s32 unk10 (j), s32 unk11 (k), s32 unk12 (l)
+ * FORMAT
+ *   aaaaaaaa bbbbbbbb cccccccc dddddddd eeeeeeee ffffffff gggggggg hhhhhhhh iiiiiiii jjjjjjjj kkkkkkkk llllllll
+ *   size = 0x30
+ */
+#define CS_UNK_DATA(unk1, unk2, unk3, unk4, unk5, unk6, unk7, unk8, unk9, unk10, unk11, unk12) \
+    CMD_W(unk1), CMD_W(unk2), CMD_W(unk3), CMD_W(unk4), CMD_W(unk5), CMD_W(unk6), \
+    CMD_W(unk7), CMD_W(unk8), CMD_W(unk9), CMD_W(unk10), CMD_W(unk11), CMD_W(unk12)
+
+/**
+ * ARGS
+ *   s32 cmdType (c), s32 entries (e)
+ * FORMAT
+ *   cccccccc eeeeeeee
+ *   size = 0x8
+ */
+#define CS_ACTOR_ACTION_LIST(cmdType, entries) CMD_W(cmdType), CMD_W(entries)
+
+/**
+ * ARGS
+ *   s16 actorAction (a), s16 startFrame (s), s16 endFrame (e),
+ *   s16 rotX (u),        s16 rotY (v),       s16 rotZ (w),
+ *   s32 startX (i),      s32 startY (j),     s32 startZ (k),
+ *   s32 endX (l),        s32 endY (m),       s32 endZ (n),
+ *   f32 normX (x),       f32 normY (y),      f32 normZ (z),
+ * FORMAT
+ *   aaaassss eeeeuuuu vvvvwwww iiiiiiii jjjjjjjj kkkkkkkk llllllll mmmmmmmm nnnnnnnn xxxxxxxx yyyyyyyy zzzzzzzz
+ *   size = 0x30
+ */
+#define CS_ACTOR_ACTION(actorAction, startFrame, endFrame, rotX, rotY, rotZ, startX, startY, startZ, endX, endY, endZ, normX, normY, normZ) \
```

Comment:
we mentioned in discord changing he Actor Action command to NPC Action, since that is the name they used and would be a little bit more specific. Im not exactly sure what would need to change where, but wanted to mention it